test/FunctionalTests/Microsoft.OData.Edm.Tests/Csdl/CsdlReaderTests.cs

```diff
@@ -412,6 +412,234 @@ public void ReadNavigationPropertyPartnerTypeHierarchyTest()
             Assert.Equal(entityTypeA3.FindProperty("A3Nav"), ((IEdmNavigationProperty)entityTypeB.FindProperty("BNav2")).Partner);
         }
 
+        [Fact]
+        public void ReadAnnotationWithoutSpecifiedValueAndUseDefault()
+        {
+            var csdl
+                = "<?xml version=\"1.0\" encoding=\"utf-16\"?>" +
+                 "<edmx:Edmx Version=\"4.0\" xmlns:edmx=\"http://docs.oasis-open.org/odata/ns/edmx\">" +
+                   "<edmx:DataServices>" +
+                     "<Schema Namespace=\"NS\" xmlns=\"http://docs.oasis-open.org/odata/ns/edm\">" +
+                       "<ComplexType Name=\"Complex\">" +
+                         "<Annotation Term=\"NS.MyAnnotationPathTerm\" AnnotationPath=\"abc/efg\" />" +
+                         "<Annotation Term=\"NS.MyDefaultTerm\" />" + // does not specify value
+                       "</ComplexType>" +
+                       "<Term Name=\"MyAnnotationPathTerm\" Type=\"Edm.AnnotationPath\" Nullable=\"false\" />" +
+                       "<Term Name=\"MyDefaultTerm\" Type=\"Edm.String\" DefaultValue=\"This is a test\" AppliesTo=\"Property Term\" Nullable=\"false\" />" +
+                     "</Schema>" +
+                   "</edmx:DataServices>" +
+                 "</edmx:Edmx>";
+            // Parse into CSDL
+            var model = CsdlReader.Parse(XElement.Parse(csdl).CreateReader());

```

Comment:
The `XmlReader` returned by `CreateReader` implements `IDisposable` and should be wrapped in a `using` block